src/pages/home/report/ReportActionsList.js

```diff
@@ -145,6 +141,10 @@ class ReportActionsList extends React.Component {
     }
 
     render() {
+        // Non-animating skeleton UI represents that chat history exists but is not loaded locally in Onyx.
```

Comment:
>Non-animating skeleton UI represents that

I'd rephrase this in this format

>When [x] happens a non-animating skeleton UI is shown because...

But honestly, I'm a bit confused about how to finish that sentence. 

I'm having trouble understanding the idea that "chat history exists but is not loaded locally in Onyx" so I think we need to improve this comment more generally. 

Does it mean that the data exists in storage, but not yet read into memory? 

Or does it mean that we know some data exists on the server, but we haven't tried to load it yet? 

Let's please try to be very specific here so that people don't have to work too hard to understand why we are doing something.

Reply:
> Or does it mean that we know some data exists on the server, but we haven't tried to load it yet?
This is the case here, I will update it.

Reply:
Awesome thanks! And to make sure I understand - we know this is the case because we have some report data already stored, the report is "not optimistic", and no report actions?

Reply:
Yep, that's my understanding so far 😄 (This is my first PR in like 4-5 months).